src/components/InstancesList/InstancesList.js

```diff
@@ -129,9 +130,23 @@ class InstancesView extends React.Component {
   }
 
   copyInstance = (instance) => {
+    const {
+      precedingTitles,
+      succeedingTitles,
+    } = instance;
     let copiedInstance = omit(instance, ['id', 'hrid']);
+
+    if (precedingTitles?.length) {
+      copiedInstance.precedingTitles = omitFromArray(precedingTitles, 'id');
+    }
+
+    if (succeedingTitles?.length) {
+      copiedInstance.succeedingTitles = omitFromArray(succeedingTitles, 'id');
+    }
+
     copiedInstance = set(copiedInstance, 'source', 'FOLIO');
-    this.setState({ copiedInstance });
+
+    if (instance?.precedingTitles) { this.setState({ copiedInstance }); }
```

Comment:
Can you explain this conditional, @mkuklis? It really changes the logic of when `setState` is called and I don't understand (a) why it's changing or (b) why, if do in fact want the conditional, why we want it for `precedingTitles` only and not `succeedingTitles` also.

Reply:
@zburke that's my mistake! Thank you for catching it.